docs/hub-tutorials/join-mainnet.md

```diff
@@ -1,130 +1,98 @@
-mv genesis.cosmoshub-4.json ~/.gaia/config/genesis.json
-```
+- [Explorers](#Explorers)
+- [Getting Started](#Getting-Started)
+- [General Configuration](#General-Configuration)
+    - [Initialize Chain](#Initialize-Chain)
+    - [Genesis File](#Genesis-File)
+    - [Seeds & Peers](#Seeds-amp-Peers)
+    - [Gas & Fees](#Gas-amp-Fees)
+    - [Pruning of State](#Pruning-of-State)
+    - [REST API](#REST-API)
+    - [GRPC](#GRPC)
+- [Choosing a Sync Mode](#Choosing-a-Sync-Mode)
+    - [Sync from Scratch (Full/Archive Node)](#Sync-from-scratch)
+    - [State Sync](#State-Sync)
+    - [Quicksync](#Quicksync)
+- [Snapshots](#Snapshots)
+- [Hardware Requirements](#Hardware)
+- [Releases](#Releases-amp-Upgrades)
+- [Cosmovisor](#Cosmovisor)
+- [Running via Background Process](#Running-via-Background-Process)
+- [Exporting State](#Exporting-State)
+- [Verify Mainnet](#Verify-Mainnet)
 
-**Go**
-Starts Gaia
-```bash
-gaiad start --p2p.seeds bf8328b66dceb4987e5cd94430af66045e59899f@public-seed.cosmos.vitwit.com:26656,cfd785a4224c7940e9a10f6c1ab24c343e923bec@164.68.107.188:26656,d72b3011ed46d783e369fdf8ae2055b99a1e5074@173.249.50.25:26656,ba3bacc714817218562f743178228f23678b2873@public-seed-node.cosmoshub.certus.one:26656,3c7cad4154967a294b3ba1cc752e40e8779640ad@84.201.128.115:26656,366ac852255c3ac8de17e11ae9ec814b8c68bddb@51.15.94.196:26656 --x-crisis-skip-assert-invariants
-```
-Now wait until the chain reaches block height 6910000. It will panic and you will see a message like:
-```
-ERR UPGRADE "Gravity-DEX" NEEDED at height: 6910000: v5.0.0-4760cf1f1266accec7a107f440d46d9724c6fd08
+### Background
 
-panic: UPGRADE "Gravity-DEX" NEEDED at height: 6910000: v5.0.0-4760cf1f1266accec7a107f440d46d9724c6fd08
-```
-Then you should run the following commands:
-```bash
-git checkout -b v5.0.5
-make install
-gaiad start --x-crisis-skip-assert-invariants --p2p.seeds bf8328b66dceb4987e5cd94430af66045e59899f@public-seed.cosmos.vitwit.com:26656,cfd785a4224c7940e9a10f6c1ab24c343e923bec@164.68.107.188:26656,d72b3011ed46d783e369fdf8ae2055b99a1e5074@173.249.50.25:26656,ba3bacc714817218562f743178228f23678b2873@public-seed-node.cosmoshub.certus.one:26656,3c7cad4154967a294b3ba1cc752e40e8779640ad@84.201.128.115:26656
-```
+The current Cosmos Hub mainnet `cosmoshub-4` has been performing in place store migration upgrades as of the [Delta Upgrade](https://github.com/cosmos/gaia/blob/main/docs/migration/cosmoshub-4-delta-upgrade.md) July 2021. The most recent upgrade was [Vega](https://github.com/cosmos/gaia/blob/main/docs/migration/cosmoshub-4-vega-upgrade.md) December 2021. This type of upgrade preserves the same chain-id but is otherwise similar to a traditional hub upgrade, meaning historical state is lost after the upgrade takes place (still accessible of course by a v4.2.1 full node). Visit the [migration section](https://github.com/cosmos/gaia/tree/main/docs/migration) of the Hub's docs for more information on previous chain migrations.
```

Comment:
Fair enough, this section was derived from material that was already there. Moving forward we'll be officially treating cosmos/mainnet as an archive repo so this might fit best there.